src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -90,6 +91,23 @@ const QUERY = gql`
             }
           }
         }
+        metadata {
+          biotype {
+            label
+            value
+            definition
+          }
+          canonical {
+            value
+            label
+            definition
+          }
+          mane {
+            value
+            label
+            definition
+          }
```

Comment:
Where do you use all this metadata for gene external references? You probably only need canonical.value and mane.value for sorting?

Reply:
yeah you are right. It is used for default sort in this component. updated